quickstart/azure/setup.md

```diff
@@ -92,17 +94,31 @@ For example, a common Service Principal as displayed by the Azure CLI looks some
 }
 ```
 
+However, you also need to obtain subscription id. To get current subscription Id you can use:
+
+```
+az account show --query id -o tsv
```

Comment:
```suggestion
$ az account show --query id -o tsv
```

Reply:
you sure thats a good idea? that `$` is really annoying, you cant copy\paste the whole line because of that

Reply:
Existing commands on this page are prefixed with `$`, so this just makes the additions consistent with the rest of the page.

I agree it makes copying a little annoying — we have plans to make copying easier via `user-select: none` applied to the `$` and providing a copy button when hovering over code snippets to quickly copy without having to select the text.